OracleDatabase/SingleInstance/README.md

```diff
@@ -456,6 +498,79 @@ Once the container has been started you can connect to it just like to any other
+
+Oracle True Cache is an in-memory, consistent, and automatically managed cache for Oracle Database.
+
+#### Setting Up Network for Communication Between Primary Database and True Cache Container
+
+* Oracle Database Free True Cache container (True Cache container) and the Oracle Database Free Primary Database container (Primary Database container) must be on the same podman network to communicate with each other.\
+Set up a podman network for inter-container communication using the following command which creates a bridge connection enabling communication between containers on the same host.
+
+        podman network create tc_net
+
+    Fetch the default subnet assigned to above network by running the following command:
+
+        podman inspect tc_net | grep -iw 'subnet'
+
+    Pick any two IP addresses from the preceding subnet and assign one for the Primary Database container (say, PRI_DB_FREE_IP) and the other for the True Cache container (say, TRU_CC_FREE_IP).
+
+    For communication across hosts, create a macvlan or ipvlan connection per [documentation](https://docs.podman.io/en/latest/markdown/podman-network-create.1.html). \
+    Specify the preceding podman network using the --net option to the podman run command of both the Primary Database container and the True Cache container as shown in following sections.
+
+#### Running Oracle Database Free True Cache in a Container
+
+* Launch the Oracle Database Free Primary Database container using the `podman run` command as follows:
+
+        podman run -td --name pri-db-free \
+        --hostname pri-db-free \
+        --net=tc_net \
+        --ip <PRI_DB_FREE_IP> \
+        -p :1521 \
+        --secret=oracle_pwd \
+        -e ENABLE_ARCHIVELOG=true \
+        -e ENABLE_FORCE_LOGGING=true \
+        -v [<host mount point>:]/opt/oracle/oradata \
+        container-registry.oracle.com/database/free:latest
+
+    Ensure that your Primary Database container is up and running and in a healthy state.
+
+    **Note:** Enable archive logging and optionally force logging in the Primary Database to support True Cache. These are not enabled by default in the prebuilt database image.
+You need to run the preceding podman run command with the host mount point (empty directory) so that a new database setup will start with these options enabled. Otherwise, you must run sql commands manually from the pri-db-free container to enable these options.
+
+* Launch the Oracle Database Free True Cache container using the `podman run` command as follows:
+
+        podman run -td --name tru-cc-free \
+        --hostname tru-cc-free \
+        --net=tc_net \
+        --ip <TRU_CC_FREE_IP> \
+        -p :1521 \
+        --secret=oracle_pwd \
+        -e TRUE_CACHE=true \
+        -e PRIMARY_DB_PWD_FILE=/var/tmp/orapwFREE \
+        -e PRIMARY_DB_CONN_STR=<PRI_DB_FREE_IP>:1521/FREE \
+        -v [<host mount point>:]/opt/oracle/oradata \
+        container-registry.oracle.com/database/free:latest
```

Comment:
For all other `docker run` instructions, we are using unqualified image names, such as `oracle/database:23.4.0-free` etc.
Is there a reason why here we are pointing to the container-registry.oracle.com image directly?

Reply:
Fixed. It was a copy/paste error